File: Molinos.Scato.Dominio/Entidades/NominacionDetalleIntervencion.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Molinos.Scato.Dominio.Entidades
{
    public class NominacionDetalleIntervencion : IIdentificable
    {
        [Key]
        public virtual int Id { get; set; }
        public virtual bool Precintado { get; set; }
        public virtual string PrecintadoACuentaDe { get; set; }
        public virtual bool DraftSurvey { get; set; }
        public virtual string SurveyACuentaDe { get; set; }
        public virtual bool PermisoDeEmbarque { get; set; }
        public virtual bool EstibadorYTrimado { get; set; }
        public virtual string Fumigacion { get; set; }
        public virtual CompaniaDeFumigacion CompaniaDeFumigacion { get; set; }
        public virtual string ACuentaDe { get; set; }
        public virtual string Observaciones { get; set; }
        public virtual TipoDeFumigacion TipoDeFumigacion { get; set; }
    }
}

File: Molinos.Scato.WebPuerto/src/app/shared/modelos/programa-embarque/nominacion-detalle-intervencion.ts
import { Destino } from "@ScatoModels/destino";
import { Exportador } from "@ScatoModels/exportador";

export class NominacionDetalleIntervencion {
    id: number;
    precintado : boolean;
    precintadoACuentaDe : string;
    draftSurvey : boolean;
    surveyACuentaDe : string;
    permisoDeEmbarque : boolean;
    estibadorYTrimado : boolean;
    fumigacion : string;
    companiaDeFumigacion : CompaniaDeFumigacion;
    companiaACuentaDe : string;
    observaciones : string;
    tipoDeFumigacion : TipoDeFumigacion;
    senasa : Senasa[];
}

export class CompaniaDeFumigacion{
    id : number;
    descripcion : string;
    mail : string;
}

export class TipoDeFumigacion{
    id : number;
    descripcion : string;
}

export class Senasa{
    id : number
    nominacionDetalleIntervencion : NominacionDetalleIntervencion;
    exportador : Exportador;
    destino : Destino;
    tieneSenasa : boolean;
    consumo : string;
    aCuentaDe : string;
    ip : boolean;
    gmo : boolean;
    fito : boolean;
    muestraOficial : boolean;
    certificadoInocuidad : boolean;
    certificadoVeterinario : boolean;
    observaciones : string;
}